test/parallel/test-buffer-readuintle.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+// testing basic functionality of readUIntLE()
+
+const buf = Buffer.from([42, 84, 168, 127]);
+const result = buf.readUIntLE(2);
+
+assert.strictEqual(result, 168);
+
+assert.throws(
+      () => {
+        buf.readUIntLE(5);
+      }, RangeError
```

Comment:
I would take it a step further and match the entire message from beginning to end, since (currently) any change to an error message is considered a breaking change.

```js
/^RangeError: Index out of range$/
```